src/Profile/ProfileKeys.hpp

```diff
@@ -94,6 +94,15 @@ extern const char HomeWaypoint[];
 extern const char HomeLocation[];
 extern const char TeamcodeRefWaypoint[];
 extern const char PilotName[];
+extern const char WeGlideEnabled[];
+extern const char WeGlidePilotID[];
+extern const char WeGlidePilotName[];     // future use
```

Comment:
We don't need keys we don't use (currently). Add them when they're implemented.
Also don't add keys without the implementation. Splitting those commits doesn't make sense; because this change is not self-contained, which a good commit must be! This commit does nothing.

Reply:
All future fields removed